src/sass/products/products.scss

```diff
@@ -0,0 +1,203 @@
+* {
+  margin: 0;
+}
+
+ul,
+ol {
+  list-style-type: none;
+  padding-left: 0;
+}
+
+img {
+  display: block;
+  width: 100%;
+  height: auto;
+}
+
+.container {
+  width: 320px;
+  display: flex;
+  align-items: center;
+  padding: 0;
+  margin-left: auto;
+  margin-right: auto;
+  justify-content: center;
+  padding-left: 20px;
+  padding-right: 20px;
+}
+
+@media screen and (min-width: 768px) {
+  .container {
+    width: 768px;
+    padding-left: 34px;
+    padding-right: 34px;
+  }
+}
+
+@media screen and (min-width: 1200px) {
+  .container {
+    width: 1200px;
+    padding-left: 115px;
+    padding-right: 115px;
+  }
+}
```

Comment:
Do tego miejsca te style znajdują się już w naszym projekcie w sass/base/_all
Jako, że cały projekt będzie korzystał finalnie z jednego css skompilowanego przez sass z różnych plików to nie ma potrzeby tego podwajać